data/announcements/scheduled.yaml

```diff
@@ -153,7 +153,7 @@ announcements:
     Please read our [announcement](/blog/2023/03/10/image-registry-redirect/) for more details.
 
 - name: Redirecting k8s.gcr.io - After
-  startTime: 2023-04-05T00:00:00 # This should run before and after Kubecon EU 2023
+  startTime: 2023-03-27T00:00:00 # This should run after the redirect begins
   endTime: 2023-05-31T00:00:00
   style: "background: #c70202"
   title: Legacy k8s.gcr.io container image registry will be redirected to registry.k8s.io
```

Comment:
this title should probably be updated as well

Reply:
And probably also some language about how it's being phased in / not complete exactly